ghostwriter-jdk-v7/src/main/java/io/ghostwriter/openjdk/v7/GhostWriterAnnotationProcessor.java

```diff
@@ -50,8 +50,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             }
         }
 
-        final boolean doClaimAnnotations = false;
-        return doClaimAnnotations;
+        return false;
```

Comment:
[This link explains it](https://docs.oracle.com/javase/8/docs/api/javax/annotation/processing/Processor.html#process-java.util.Set-javax.annotation.processing.RoundEnvironment-)

I used the variable to "document" why we return false. We do not want to claim any annotation with GW, this way it plays nicely together with other annotation processors.

https://docs.oracle.com/javase/8/docs/api/javax/annotation/processing/Processor.html#process-java.util.Set-javax.annotation.processing.RoundEnvironment-